pkg/util/utils.go

```diff
@@ -106,3 +106,36 @@ func (m MultiError) ToError() error {
 	}
 	return fmt.Errorf(strings.Join(errStrings, "\n"))
 }
+
+func VersionOrdinal(version string) string {
+	// ISO/IEC 14651:2011
+	// https://www.iso.org/standard/57976.html
+	// This method is applicable for 255 characters string
+	// to determine their collating order in a sorted list.
+	// It create a collating sorted list and return the string.
```

Comment:
I think we should ass comments before the function, so that it would be visible in https://godoc.org/github.com/minishift/minishift